runtime/vm/object.cc

```diff
@@ -19334,12 +19334,9 @@ RawString* Number::ToString(Heap::Space space) const {
 
 
 const char* Double::ToCString() const {
```

Comment:
This function is called directly from the C++ code bypassing Dart code so this change is not correct.